### PR TITLE
feat(ecl-compliance): Card, adding child template - FRONT-1058

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: src/ec/packages/ec-component-card/card.story.js [hunk 1]
```diff
@@ -90,27 +92,27 @@ const formatCard = data => {
...
   // Meta.
   const meta = data.meta
-    ? array('card.meta (tags)', data.meta, '|', optionalGroupId)
+    ? array('card.meta', data.meta, '|', tabLabels.optional)
     : false;
 
   const formatTag = (tag, index) => {
     tag.label = text(
-      `card.tags.label (Tag ${index})`,
+      `card.tags[${index}].label`,
```
Comment (kalinchernev) [inline src/ec/packages/ec-component-card/card.story.js:113]: Why making the index a string here? It doesn't seem necessary.

Comment (planctus) [reply src/ec/packages/ec-component-card/card.story.js:113]: To get this, no? 
image
we use the same also for infos and links

**Comment by planctus** [comment]
> "and none of" part is not a clear message.

I really welcome any proposal in the way synctatically or semantically write this messages, sometimes it's not easy and this one in link was the hardest.

The fact that you get the warning message from the link component is correct in the current approach, while implementing the card that way you are also wrongly implementing a "link" and you get "potentially" additional info from that component.
The one i expressed with a certain difficulty, is indeed the only condition where we define the link as not compliant. The logic behind that is that when you implement a link you could have an empty label, for instance, but then we would expect an icon, maybe it's too verbose the way it is, but i thought it needed these info to be understood, let's make it better if it's not clear.

File: src/ec/packages/ec-component-card/card.story.js (hunk 1 above)
Comment (kalinchernev) [reply src/ec/packages/ec-component-card/card.story.js:113]: oh, sorry, it's the whole link, it's correct!

**Comment by kalinchernev** [comment]
> while implementing the card that way you are also wrongly implementing a "link" and you get "potentially" additional info from that component.
I haven't implemented anything here

The suggestion for making the pattern more clear is being already discussed at https://github.com/ec-europa/ecl-twig/pull/437#discussion_r425091653

**Pull request merged**